toolkit/toolkit/about/aboutGlean.ftl

```diff
@@ -141,6 +142,35 @@ about-glean-about-data-list-item-debug-ping-viewer =
 about-glean-about-data-list-item-firefox-profiler =
     To record a performance profile and see all recorded metrics, please use the
     <a data-l10n-name="about-glean-firefox-profiler">{ -profiler-brand-name }</a>.
+about-glean-metrics-table-header = All Metrics
+# This message refers to the category in which a given metric is recorded.
+about-glean-metrics-table-header-category = Category
+# This message refers to the name of a given metric.
+about-glean-metrics-table-header-name = Name
+# This message refers to a given metric's metric type.
+about-glean-metrics-table-header-type = Type
+# This message refers to the underlying value of a given metric.
+about-glean-metrics-table-header-value = Value
+# This message refers to the UI action buttons for a given metric.
+about-glean-metrics-table-header-actions = Actions
+about-glean-label-for-filter-metrics = Filter
```

Comment:
This will need a comment, no idea if it's a verb or a noun without seeing the UI.